content/acra/security-controls/key-management/operations/rotation.md

```diff
@@ -123,14 +129,31 @@ Acra integration into existing infrastructure, since no changes to application s
 
 ## How to rotate Acra keys
 
-`acra-keymaker` tool can be used to generate new keys.
+[`acra-keymaker`](/acra/configuring-maintaining/general-configuration/acra-keymaker/) tool can be used to generate new keys.
 After that, you will need to distribute the new public keys.
 Data storage keys may require additional attention.
 
 Make sure you have set up the correct master key before using `acra-keymaker`.
 Please refer to the [key generation instructions](../generation/#master-keys)
 to learn about master keys.
 
+
+### Rotating Acra Master key
+
+There’s no one single button to rotate Acra master key, but it’s possible using Acra tools. It could be done manually or automated.
+
+The easiest way is to use [acra-backup utility](/acra/configuring-maintaining/general-configuration/acra-backup/):
+
+- Stop generating new intermediate keys.
+- Call `acra-backup` to `export` all keys from keystore using current `ACRA_MASTER_KEY`. All keys will be encrypted with temporary `BACKUP_MASTER_KEY` generated by `acra-backup`, save this key.
+- Generate new rotated ACRA_MASTER_KEY.
```

Comment:
we can mention that users should or may use [acra-keys](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-keys/generate/#acra-master-key)/[acra-keymaker](https://docs.cossacklabs.com/acra/configuring-maintaining/general-configuration/acra-keymaker/#acra-master-key). It much better because we require master key with known 32byte length. Plus it is simple master key for keystore v1 and more complicated for v2 (json). So much better to park users to our tools here too)

Reply:
good idea, done